src/applications/appeals/10182/content/saveInProgress.js

```diff
@@ -15,8 +15,8 @@ export const startText = 'Start the Board Appeal request';
 export const unauthStartText = 'Sign in to start your request';
 
 export const savedFormMessages = {
-  notFound: 'Please start over to apply for a board appeal',
-  noAuth: 'Please sign in again to continue your request for a board appeal',
+  notFound: 'Start over to apply for a board appeal',
+  noAuth: 'Sign in again to continue your request for a board appeal',
```

Comment:
Is sign-in not considered required in this case in the context of when to use `Please`?

Reply:
I copied this from the content source of truth, I think (not on my work computer now)... I noticed that all usage of "please" was removed.